### ISSUE TITLE
Error main:186 Failed to install

### ISSUE THREAD

**Issue by xFahrenheightx** [angelXwind/AppSync#29]
Trying to install a gameboy emulator for my Apple Watch on Electra on 11.3.1. I get this error with every app I try to install. Any help would be appreciated.

2018-07-07 01:49:08.894 appinst[11984:213221] appinst (App Installer)
2018-07-07 01:49:08.896 appinst[11984:213221] Copyright (C) 2014-2018 Linus Yang, Karen/あけみ (angelXwind)
2018-07-07 01:49:08.897 appinst[11984:213221] ** PLEASE DO NOT USE APPINST FOR PIRACY **
2018-07-07 01:49:08.908 appinst[11984:213221] appinst: main:58 Cleaning up temporary files...
2018-07-07 01:49:08.925 appinst[11984:213221] appinst: main:133 Installing com.gabrieloc.giovanni ...
2018-07-07 01:49:10.032 appinst[11984:213221] appinst: main:186 Failed to install com.gabrieloc.giovanni

**Comment by angelXwind** [comment]
AppSync Unified is unable to install unsigned/fakesigned apps on the Apple Watch, as it cannot patch the verification checks on the Apple Watch, only the iPhone.

In order to achieve what you are trying to do, a jailbreak for the Apple Watch would be required (which does not yet exist, at least not in a form ready for public consumption).

tl;dr: Not a bug with AppSync Unified.

**Comment by xFahrenheightx** [comment]
Ah damn. Thank you for the response.
